strap.py

```
@@ -36,6 +36,7 @@
 version = ""
 install_to = "rootfs/filesystem"
 package = input()
+open("rootfs/filesystem/etc/redrose-strap", "a").write(package + "\n")
```

Ruff (0.15.12)

[warning] Use a context manager for opening files

(SIM115)